crates/turborepo-globwatch/src/lib.rs

```diff
@@ -287,6 +289,16 @@ impl<T: Watcher> WatchConfig<T> {
             .map_err(ConfigError::WatchError)
     }
 
+    /// Register a single path to be included by the watcher.
+    pub async fn include_path(&self, path: &Path) -> Result<(), ConfigError> {
+        trace!("watching {:?}", path);
+        self.watcher
+            .lock()
+            .expect("watcher lock poisoned")
+            .watch(path, notify::RecursiveMode::NonRecursive)
```

Comment:
I changed this to be `NonRecursive` as I don't think we want to recursively watch the repo root: @arlyon